### ISSUE TITLE
openapicmd call issue with param ?

### ISSUE THREAD

**Issue by dacoffey** [anttiviljami/openapicmd#43]
i am trying to check out this project but having problem trying to call petstore example with a param, no matter what command line arguments i try the outgoing params:{} always stays empty, any ideas?

`
API=https://petstore3.swagger.io/api/v3/openapi.json ; npx -y openapicmd call -V -v -i --operation findPetsByStatus --param=status=sold              

{
  "method": "get",
  "url": "/pet/findByStatus",
  "params": {},
  "headers": {
    "Accept": "application/json, text/plain, */*",
    "Content-Type": "text/plain"
  }
}
findPetsByStatus
GET /pet/findByStatus
RESPONSE META:
{
  "code": 400,
  "status": "Bad Request",
  "headers": {
    "date": "Sun, 17 Dec 2023 02:49:33 GMT",
    "content-type": "application/json",
    "content-length": "30",
    "connection": "close",
    "access-control-allow-origin": "*",
    "access-control-allow-methods": "GET, POST, DELETE, PUT",
    "access-control-allow-headers": "Content-Type, api_key, Authorization",
    "access-control-expose-headers": "Content-Disposition",
    "server": "Jetty(9.4.9.v20180320)"
  }
}
RESPONSE BODY:
"No status provided. Try again?"
`

**Comment by dacoffey** [comment]
something else to note, if i downgrade to 1.8.2 it works but trying 1.14.0 fails 🤔

API=https://petstore3.swagger.io/api/v3/openapi.json ; npx -y openapicmd@1.14.0 call $API -i -o findPetsByStatus -p status=sold

findPetsByStatus
GET /pet/findByStatus
RESPONSE META:
{
  "code": 400,
  "status": "Bad Request",
  "headers": {
    "date": "Sun, 17 Dec 2023 03:11:22 GMT",
    "content-type": "application/json",
    "content-length": "30",
    "connection": "close",
    "access-control-allow-origin": "*",
    "access-control-allow-methods": "GET, POST, DELETE, PUT",
    "access-control-allow-headers": "Content-Type, api_key, Authorization",
    "access-control-expose-headers": "Content-Disposition",
    "server": "Jetty(9.4.9.v20180320)"
  }
}
RESPONSE BODY:
"No status provided. Try again?"
                                                                                                        

% SAT:231216:221122 user@workbox-panera:/DEV/LAB/_LAB_/api
» API=https://petstore3.swagger.io/api/v3/openapi.json ; npx -y openapicmd@1.8.2 call $API -i -o findPetsByStatus -p status=sold

findPetsByStatus
GET https://petstore3.swagger.io/api/v3/pet/findByStatus?status=sold
RESPONSE META:
{
  "code": 200,
  "status": "OK",
  "headers": {
    "date": "Sun, 17 Dec 2023 03:11:26 GMT",
    "content-type": "application/json",
    "content-length": "1087",
    "connection": "close",
    "access-control-allow-origin": "*",
    "access-control-allow-methods": "GET, POST, DELETE, PUT",
    "access-control-allow-headers": "Content-Type, api_key, Authorization",
    "access-control-expose-headers": "Content-Disposition",
    "server": "Jetty(9.4.9.v20180320)"
  }
}
RESPONSE BODY:
[
  {
    "id": 5,
    "category": {
      "id": 1,
      "name": "Dogs"
    },
    "name": "Dog 2",
    "photoUrls": [
      "url1",
      "url2"
    ],
    "tags": [
      {
        "id": 1,
        "name": "tag2"
      },
      {
        "id": 2,
        "name": "tag3"
      }
    ],
    "status": "sold"
  },
  {
    "id": 9223372016900036000,
    "category": {
      "id": 81261053,
      "name": "Bengal"
    },
    "name": "Milo",
    "photoUrls": [
      "https://example.com/image1.jpg"
    ],
    "tags": [
      {
        "id": 308668,
        "name": "Chartreux"
      },
      {
        "id": 14194199,
        "name": "Shiba Inu"
      }
    ],
    "status": "sold"
  },
  {
    "id": 9223372016900081000,
    "category": {
      "id": 8784288,
      "name": "Saint Bernard"
    },
    "name": "Milo",
    "photoUrls": [
      "https://example.com/image1.jpg"
    ],
    "tags": [
      {
        "id": 416352,
        "name": "Cocker Spaniel"
      },
      {
        "id": 17203042,
        "name": "Shiba Inu"
      }
    ],
    "status": "sold"
  },
  {
    "id": 9223372016900082000,
    "category": {
      "id": 3711228,
      "name": "Himalayan"
    },
    "name": "Sadie",
    "photoUrls": [
      "https://example.com/image1.jpg"
    ],
    "tags": [
      {
        "id": 527127,
        "name": "Egyptian Mau"
      },
      {
        "id": 21144561,
        "name": "Cocker Spaniel"
      }
    ],
    "status": "sold"
  },
  {
    "id": 9223372016900041000,
    "category": {
      "id": 73470798,
      "name": "Savannah"
    },
    "name": "Charlie",
    "photoUrls": [
      "https://example.com/image1.jpg"
    ],
    "tags": [
      {
        "id": 67873,
        "name": "French Bulldog"
      },
      {
        "id": 4672546,
        "name": "Bichon Frise"
      }
    ],
    "status": "sold"
  }
]

**Comment by anttiviljami** [comment]
Hi @dacoffey! Thanks for reporting this. The issue is fixed in `openapicmd@2.2.0`:

```
$ npx -y openapicmd call https://petstore3.swagger.io/api/v3/openapi.json -o findPetsByStatus -p status=sold -vi

REQUEST META:
{
  "operationId": "findPetsByStatus",
  "method": "get",
  "url": "https://petstore3.swagger.io/api/v3/pet/findByStatus?status=sold",
  "path": "/pet/findByStatus",
  "pathParams": {},
  "query": {
    "status": "sold"
  },
  "queryString": "status=sold",
  "headers": {
    "Accept": "application/json, text/plain, */*"
  },
  "cookies": {}
}
RESPONSE META:
{
  "code": 200,
  "status": "OK",
  "headers": {
    "date": "Sat, 27 Jan 2024 10:31:23 GMT",
    "content-type": "application/json",
    "content-length": "1148",
    "connection": "keep-alive",
    "access-control-allow-origin": "*",
    "access-control-allow-methods": "GET, POST, DELETE, PUT",
    "access-control-allow-headers": "Content-Type, api_key, Authorization",
    "access-control-expose-headers": "Content-Disposition",
    "server": "Jetty(9.4.9.v20180320)"
  }
}
RESPONSE BODY:
[
  {
    "id": 5,
    "category": {
      "id": 1,
      "name": "Dogs"
    },
    "name": "Dog 2",
    "photoUrls": [
      "url1",
      "url2"
    ],
    "tags": [
      {
        "id": 1,
        "name": "tag2"
      },
      {
        "id": 2,
        "name": "tag3"
      }
    ],
    "status": "sold"
  },
  {
    "id": 86,
    "name": "Special_char_owner_!@#$^&()`.testing",
    "photoUrls": [
      "https://petstore3.swagger.io/resources/photos/623389095.jpg"
    ],
    "tags": [],
    "status": "sold"
  },
  {
    "id": 867,
    "name": "見響见响仮仏!$%&()*+,-ƄƅƆḞḟṀʶʷʸăĄą",
    "photoUrls": [
      "https://petstore3.swagger.io/resources/photos/623389095.jpg"
    ],
    "tags": [],
    "status": "sold"
  },
  {
    "id": 6,
    "name": "Special_char_owner_!@#$^&()`.testing",
    "photoUrls": [
      "https://petstore3.swagger.io/resources/photos/623389095.jpg"
    ],
    "tags": [],
    "status": "sold"
  },
  {
    "id": 134674,
    "name": "見響见响仮仏!$%&()*+,-ƄƅƆḞḟṀʶʷʸăĄą",
    "photoUrls": [
      "https://petstore3.swagger.io/resources/photos/623389095.jpg"
    ],
    "tags": [],
    "status": "sold"
  },
  {
    "id": 13467,
    "name": "見響见响仮仏!$%&()*+,-ƄƅƆḞḟṀʶʷʸăĄą",
    "photoUrls": [
      "https://petstore3.swagger.io/resources/photos/623389095.jpg"
    ],
    "tags": [],
    "status": "sold"
  },
  {
    "id": 2,
    "category": {
      "id": 2,
      "name": "Chaucito"
    },
    "name": "Gatitotototo",
    "photoUrls": [],
    "tags": [
      {
        "id": 1,
        "name": "Bizcocho"
      }
    ],
    "status": "sold"
  }
]
```
